src/Compilers/Test/Core/Platform/CoreClr/CoreCLRRuntimeEnvironment.cs

```diff
@@ -106,9 +115,6 @@ public int Execute(string moduleName, string[] args, string expectedOutput)
 
         public void Verify(Verification verification)
```

Comment:
>Verify [](http://example.com/codeflow?start=20&length=6)

Is this method needed?

Reply:
Yes, it implements `IRuntimeEnvironment`